TLM/TLM/Manager/Impl/LaneConnectionManager.cs

```diff
@@ -410,8 +413,6 @@ public class LaneConnectionManager
             }
         }
 
-        protected override void HandleValidSegment(ref ExtSegment seg) { }
-
         internal bool GetLaneEndPoint(ushort segmentId,
```

Comment:
this is not used in lane connection manager at all. can we move this out of here? it can go to your lane manager, laneutil, or lane connector tool.

Reply:
Look again. It's already removed. Since `LaneConnectionManager` is no longer an `AbstractGeometryObservingManager`, the method is simply gone.

Reply:
I am talking about GetLaneEndPoint()

Reply:
> I am talking about GetLaneEndPoint()

https://github.com/CitiesSkylinesMods/TMPE/blob/cb97110717a8b4a98a453d1cd777f4b67fad7a49/TLM/TLM/UI/SubTools/LaneConnectorTool.cs#L1196

Reply:
Ahhhh, okay, that makes more sense.

It's possible that this belongs in the upcoming `LaneEndManager`, but I think we want to wait until we have identified potential use cases outside of the Lane Connector, and use that as a basis to either generalize its behavior or else demonstrate that it is already sufficiently generalized.

Reply:
```cs
GetLaneEndPoint(
                            segmentId: segmentId,
                            startNode: startNode,
                            laneIndex: laneIndex,
                            laneId: laneId,
                            laneInfo: laneInfo,
                            outgoing: out bool isSource,
                            incoming: out bool isTarget,
                            pos: out Vector3 pos)
```
I had another look at this method and I think its so special purpose that it only belongs to lane connector tool.

Reply:
at least make it static.

Reply:
I think a change of this kind makes more sense in the context of the work you're doing. It has no relationship to this PR other than happening to touch one of the same classes.